postgres/datadog_checks/postgres/postgres.py

```diff
@@ -418,6 +418,12 @@ def _get_pg_attrs(self, instance):
         # Let's use pg8000
         return pg8000.connect, pg8000.InterfaceError, pg8000.ProgrammingError
 
+    def _get_replication_role(self, key, db):
+        cursor = db.cursor()
+        cursor.execute('SELECT pg_is_in_recovery();')
+        role = cursor.fetchone()[0]
+        return "standby" if role else "master"
```

Comment:
if we're a standby, `role` would be `f` here right? If thats the case `role` would always evaluate to True and get `standby`. Maybe:
`return "standby" if role == 't' else "master"`?

Reply:
per checking, the value returned for  `role` is of type bool. see below when I temporarilly added a debug line to print `type(role)`

```
2019-02-09 03:39:47 UTC | DEBUG | (datadog_agent.go:153 in LogMessage) | (postgres.py:425) | DD role v: False
2019-02-09 03:39:47 UTC | DEBUG | (datadog_agent.go:153 in LogMessage) | (postgres.py:426) | DD type(role): <type 'bool'>
```

added a comment/note to mention that.